contracts/integrations/test/coordinator/coordinator_test.ts

```diff
@@ -204,7 +137,14 @@ blockchainTests.resets('Coordinator integration tests', env => {
                     .executeTransaction(transaction, taker.address, transaction.signature, [approval.signature])
                     .awaitTransactionSuccessAsync({ from: taker.address, value: DeploymentManager.protocolFee });
 
-                const expectedBalances = await simulateFillsAsync([order], txReceipt, DeploymentManager.protocolFee);
+                const expectedBalances = LocalBalanceStore.create(balanceStore);
+                expectedBalances.simulateFills(
+                    [order],
+                    taker.address,
+                    txReceipt,
+                    deployment,
```

Comment:
Could we give `LocalBalanceStore` the `DeploymentManager` at construction? This would save some code duplication in this file

Reply:
I've thought about this, but I'd prefer to keep them decoupled for the time being